HISTORY.md

```diff
@@ -1,3 +1,16 @@
+## 1.0.1 (September 20, 20023)
```

Comment:
Given this is adding new functionality (i.e. finding and running workflows) I think this should be version `1.1.0`:

```
Given a version number MAJOR.MINOR.PATCH, increment the:

MAJOR version when you make incompatible API changes
MINOR version when you add functionality in a backward compatible manner
PATCH version when you make backward compatible bug fixes
```